react/features/notifications/components/Notification.web.js

```diff
@@ -155,11 +167,25 @@ class Notification extends Component {
     _mapAppearanceToButtons() {
         switch (this.props.appearance) {
```

Comment:
The error and warning use case are exactly the same except the text for the OK/Dismiss button, right? Could you please remove the code duplication?

Reply:
It is changed in the next commits! Please ignore this comment!